### ISSUE TITLE
use server of docker factorio instance

### ISSUE THREAD

**Issue by mok-liee** [OpenFactorioServerManager/factorio-server-manager#361]
hi,

is it possible to manage an docker instance of factorio (`image: factoriotools/factorio`) which is in the same stack (via `docker compose`)?

greetings mok

**Comment by knoxfighter** [comment]
No, our docker image does contain factorio already, there is no need for another container. You can use our docker-compose file to simply start it.
